Tools/autotest/default_params/realflight-titan-cobra.parm

```diff
@@ -0,0 +1,100 @@
+# Titan Cobra RealFlight ArduPilot 4.5
+AHRS_EKF_TYPE,3
```

Comment:
This is obviously the default value; please strip out all default values from in here.

Reply:
It's actually not the default. The default is 10 (SITL)

HOWEVER, I should be using the default of 10. Don't know when I slipped that in there on accident.